frontend/words.json

```diff
@@ -1,7 +1,7 @@
SE, REITEN, RELATION, RENNEN, RICHTIG, RIEGEL, RIEMEN, RIESE, RING, RIPPE, RITTER, ROBOTER, ROCK, ROLLE, ROLLSTUHL, ROLLTREPPE, ROM, RÖMER, ROSE, ROSINE, ROST, ROT, ROULETTE, RÜCKEN, RÜCKGRAT, RUCKSACK, RUDER, RUND, RUNDE, RUTE, SAISON, SAITE, SALON, SALZ, SAMMLUNG, SANDBURG, SANDKASTEN, SANDWICH, SÄNGER, SATELLIT, SATURN, SATZ, SAUM, SCHACH, SCHADEN, SCHAF, SCHAFE, SCHALE, SCHALTER, SCHÄRPE, SCHATZ, SCHAUFEL, SCHEIBE, SCHEIN, SCHELLE, SCHEUERN, SCHIFF, SCHIFFBRUCH, SCHILD, SCHIMMEL, SCHIMPANSE, SCHIRM, SCHLAF, SCHLAFEN, SCHLAMM, SCHLANGE, SCHLAUCH, SCHLEIER, SCHLEIFE, SCHLENDRIAN, SCHLIPS, SCHLOSS, SCHLUCK, SCHLUMMERN, SCHLÜSSEL, SCHMUTZIG, SCHNABELTIER, SCHNEE, SCHNEEBESEN, SCHNEEMANN, SCHNELL, SCHNUR, SCHNÜRSENKEL, SCHOKOLADE, SCHÖNHEIT, SCHONZEIT, SCHOTTEN, SCHRAUBE, SCHREIBTISCH, SCHRUMPFEN, SCHUH, SCHULE, SCHUPPEN, SCHURKE, SCHWAMM, SCHWARM, SCHWEISSER, SCHWIEGERMUTTER, SCHWIMMEN, SCHWINDEL, SCHWINGEN, SEGEL, SEITE, SEKRETÄR, SET, SHAKESPEARE, SHAMPOO, SHERIFF, SICHER, SICHERHEIT, SICHTWEISE, SIEB, SIEGEL, SIESTA, SIGNAL, SILHOUETTE, SINN, SKELETT, SKI, SOFA, SOFTDRINK, SOLDAT, SONNE, SONNENBRAND, SOUVENIR, SOZIALISMUS, SPEED, SPIEGEL, SPIEL, SPIELPLATZ, SPINNE, SPION, SPIONIEREN, SPITZE, SPORTLER, SPRINGER, SPROSSE, SPRUDELN, SPRUNG, SPUCKE, SPULE, SPÜLUNG, STAAT, STAB, STACHEL, STADION, STAMM, STAND, STAR, STARK, STAUB, STECKEN, STEHEND, STEIN, STEMPEL, STEPPE, STERN, STERNWARTE, STEUER, STIFT, STIMME, STIRN, STIRNRUNZELN, STOCK, STOFF, STRAND, STRASSE, STRAUSS, STREAM, STREICHHOLZ, STREIFEN, STROHHALM, STROM, STRUDEL, STUDENT, STUHL, STUMM, STUMPF, STURM, SÜDEN, SUMME, SUMPF, SUPERHELD, SUPPE, SUSHI, SÜSSIGKEITEN, TAFEL, TAG, TAGESZEIT, TAIGA, TAKT, TANTE, TANZ, TASCHE, TASCHENLAMPE, TASSE, TASTE, TAU, TAUCHER, TAXI, TEEKANNE, TEELÖFFEL, TEENAGER, TEIL, TELEFON, TELESKOP, TELLER, TEMPO, TENNIS, THEATER, TIEF, TIER, TIGER, TISCH, TOAST, TOD, TOILETTE, TOKIO, TON, TÖNUNG, TOR, TORNADO, TORTE, TRAINIEREN, TRAKTOR, TRAUBE, TRAUER, TRAUM, TRÄUMEN, TRAURIG, TREIBEN, TREIBSAND, TREPPE, TRICHTER, TRINKEN, TROCKNER, TROLL, TROMPETE, TROPFEN, TRUMPF, TUBA, TÜR, TURM, TURNIER, TÜTENSUPPE, TUTOR, ÜBERSCHRIFT UHR, UHR, UHRWERK, UMARMUNG, UMHANG, UMLAUFBAHN, UMZUG, UNSCHÄRFE, UNTERSCHRIFT, UNTERSETZER, VATER, VEGAN, VENTILATOR, VENUS, VERBAND, VERBINDUNG, VEREIN, VERENGUNG, VERGELTUNG, VERHANDELN, VERLETZT, VERRÜCKT, VERSPRECHEN, VERWIRRUNG, VERZÖGERUNG, VIERTEL, VISION, VOGEL, VORHANG, VORRICHTUNG, VORSATZ, WACHS, WAL, WALD, WAND, WANDEL, WANNE, WANZE, WÄSCHEREI, WASHINGTON, WASSER, WASSERMELONE, WATT, WEDELN, WEIDE, WEISS, WEIZEN, WELLE, WELPE, WELT, WERBEN, WERKZEUG, WERT, WESTE, WILLE, WIND, WINDMÜHLE, WINKEL, WINNETOU, WINTER, WIRTSCHAFT, WISCHMOPP, WITZ, WOLF, WOLKENKRATZER, WOLLE, WÖRTERBUCH, WUNSCH, WURF, WÜRFEL, WURM, WURZEL, ZAHN, ZAHNARZT, ZAHNBÜRSTE, ZAHNRAD, ZAR, ZAUN, ZEHENSPITZEN, ZEHN, ZEICHNEN, ZEIT, ZEITUNG, ZELLE, ZEN, ZENTAUR, ZIEL, ZIRKUS, ZITRONE, ZOLL, ZOLLSTOCK, ZONE, ZOO, ZUCKER, ZUERST, ZUFÄLLIG, ZUG, ZWEIG, ZWERG, ZYLINDER",
```

Comment:
will words with spaces between them (like `NEW YORK`) work?

Reply:
Yes this will work. I have tested it.